tinygrad/codegen/uops.py

```diff
@@ -289,5 +289,6 @@ def flops_mem(uops:List[UOp], ignore_indexing=False) -> Tuple[sint, sint]:
       flops += (mults * (2 if u.arg == TernaryOps.MULACC else 1)) * u.dtype.count
     elif u.op is UOps.WMMA and u not in dont_count:
       assert u.arg[1] is not None
-      flops += 2 * prod(u.arg[1]) // 32 * mults
+      if u.arg[4] == "INTEL": flops += 2 * prod(u.arg[1]) // 8 * mults
```

Comment:
This shouldn't be here

Reply:
I want to bundle the warp size into the wmma arg when i'm creating the lazyop so that we can multiply by that instead of something hardcoded. Device intel is specified manually for the python renderer, so changing the intel renderer device to GPU shouldn't have problems for the python emulations. wmma_arg is indexed in a strange way in some places, so I've been trying to figure out how to bundle that in without random checks breaking. May require some minor changes to some asserts